### ISSUE TITLE
Encryption by default

### ISSUE THREAD

**Issue by JJF2** [WHAnonymous/Chat-API#1335]
Starting from this morning seems that WA forced all to encryption!

There is any way to get encryption working with chatapi in windows?

**Comment by 0xTryCatch** [comment]
What do you mean forced?

**Comment by bobvandevijver** [comment]
I have the same problem: this is related to #1333. A little log:

```
rx  <message from="31645719540@s.whatsapp.net" type="text" id="1451385387.--186" t="1451385382" notify="">
�-�l�k٨��X�2�>OƸ�=�V�="pkmsg">3!��
n!�1�yL�X���>��Y��&m�K��҇�+"B3
!��}^��?
                u�<�*ٺ/����k���{"�A�񵎊lV�Ñ������:�(����0߱�</enc>
rx  </message>
```

This was not encrypted before, and now my bot can no longer receive anything (sending works by the way).

**Comment by JJF2** [comment]
I mean that ALL the received message become suddently encrypted by this morning at 9AM CET.
Also contacts that before 9 sent plain text messages now send only enc.

I'm using chatapi on windows and I didn't find a way to install crypto extension.

The unreceived encrypted messages cause WA to throw stream:error after exactly 60s.

When I restart my daemon, all the enc messages are received again and again.

In short, chatapi without crypto extension seem dead

**Comment by JJF2** [comment]
```xml
29-12-15 12:01:44pm - rx  <message from="393*********@s.whatsapp.net" type="text" id="2C8417EBB**********" offline="3" t="1451380774" notify="NAME">
rx    <enc v="1" type="pkmsg">3! 1ÂºXiQÉK°ÿ§1ÜW€»·+Íˆñá§×_U·K!ÁYüˆ¼=ãB»+»,á'·¦šàüxù¬€Óƒl"B3!º—,p®0Òüè/Ó—­êÌÜ/w?‚QëÚÔÇQº‡v  "žl;â`—ñ¹¾’5…©ãb›ªk~õx(´œøÖ0ã‘î</enc>
rx  </message>

29-12-15 12:01:45pm - rx  <message from="393*********@s.whatsapp.net" type="text" id="65D1968D***********" offline="3" t="1451382661" notify="NAME">
rx    <enc v="1" type="pkmsg">3!¤ô¤}MÒ¨aW·Ú%ŸfR ’O¿è]ŽùQÕº !l¡ðw ùu¥­§¡ouO”£·Ö£ØÝöTÊ£»ð~"B3!ÎðzQ;…ÌZKæÞ¦ â’•Ò¤šrÊ‚@º›…0  "ä·=+T.Ä¥>¤¢åxCÁèZ+ä×(‰ ›Ö0ì—</enc>
rx  </message>

29-12-15 12:01:46pm - rx  <message from="393*********@s.whatsapp.net" type="text" id="BAEB8C16***********" offline="2" t="1451383292" notify="NAME">
rx    <enc v="1" type="pkmsg">3!þ&à²ÔŸ°uK§.„Åˆh?`nN?lï¶¤›I!¸Û³ð»=ÇÕ\˜:BÕãc.kbÛÀûO‚Êój"b3!(xB‹@É­­ö3À`ÀSà)TVùaã\˜K¼|•0  "08aøPõË¥,0He»Co©}¶±í¿Ê‹,‹ }úößl	†c²ONzlÇnîC(YbU(¢N0þè²</enc>
rx  </message>

29-12-15 12:01:47pm - rx  <message from="393*********@s.whatsapp.net" type="text" id="C1BCC023***********" offline="2" t="1451383740" notify="NAME">
rx    <enc v="1" type="pkmsg">3!Ÿ±!Ù‹;“À‰1gªU.yboAáÃ)÷€ò*NøF{!SI §Yÿî¸),á*A!ûOEªôCÇrˆ¡¦Ù–‰N"B3!bjéÓ3U"Ï=\fÙ®óq}KïqÍV$#'»N>ƒ\g  "­W´þ.\¯¬t¿ïARh{O3÷Êh(—¨õÛ0¤µ¿ </enc>
rx  </message>

29-12-15 12:01:48pm - rx  <ib from="s.whatsapp.net">
rx    <offline count="4"></offline>
rx  </ib>

29-12-15 12:01:49pm - rx  <presence from="393*********@s.whatsapp.net"></presence>

29-12-15 12:01:50pm - rx  <presence from="393*********@s.whatsapp.net"></presence>

29-12-15 12:02:50pm - rx  <stream:error>
rx    <ack id="2C8417EBB**********"></ack>
rx  </stream:error>
```
